packages/neos-ui/src/Sagas/Changes/index.js

```diff
@@ -9,6 +9,12 @@ const {publishableNodesInDocumentSelector, baseWorkspaceSelector} = selectors.CR
 function * persistChanges(changes) {
     const {change} = backend.get().endpoints;
 
+    const confirmExit = () => {
```

Comment:
Needs to be I18n

Reply:
Or "are currently being persisted".
In any case, nobody would see this message in modern browsers, since they replace it with generic text :)

Reply:
Thinking about it more, it might not worth the fuss to do i18n, and just rely on the default browser's text for the alert. I think only IE supports custom text for those alerts.